tests/e2e/test_rosacli_operator_roles.go

```diff
@@ -49,7 +50,60 @@ var _ = Describe("Edit operator roles", labels.Feature.OperatorRoles, func() {
 			err := rosaClient.CleanResources(clusterID)
 			Expect(err).ToNot(HaveOccurred())
 		})
+		It("to delete in-used operator-roles and byo oidc-config  [id:74761]",
+			labels.Critical, labels.Runtime.OCMResources, func() {
```

Comment:
cases labeled with `labels.Runtime.OCMResources` should can run independently. That means it is not running based on a created cluster.